test/mmap.cxx

```diff
@@ -103,7 +104,6 @@ TEST_CASE("Structure serialization and loading", "[mmap_t]")
 		// REQUIRE(std::memcmp(_d, &d, sizeof(foo)) == 0);
 	}
 
-
-
+	static_cast<void>(remove("mmap_t.serialized"));
```

Comment:
As we spoke in private chat, Catch2 doesn't have a strict total order of sections, so this will actually be executed as part of each section's setup code.

I think you can instead wrap the fd_t into a naughtyPtr_t as I did back for $work.

Reply:
This should now be fixed - thank you for the suggestion